main/activity-log.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+
+const Store = require('electron-store')
+const configStore = new Store({
```

Comment:
```suggestion
const activityLogStore = new Store({
```

would this be a more fitting name?